index.html

```diff
@@ -5,9 +5,143 @@
+    </nav>
+    <header>
+      <h2>"Ready for First Contact!"</h2>
+      <h1>FIRST INTERPLANETARY ARRIVAL CONFERENCE</h1>
+      <div>
+        <p>
+          Our first ever contact with extraterrestrial beings. They’ve heard us,
+          we’ve been contacted... Now it's time to meet them. They say they mean
+          no harm and only want to be heard
+        </p>
+      </div>
+      <h3>2022.06.25(SAT) ~ 26(SUN)</h3>
+      <h4>&commat; National Comission of Interplanetary Affairs</h4>
+    </header>
+    <section class="program" id="program">
+      <h2>Main Program</h2>
+      <hr />
+      <div class="program-cards">
+        <div class="card">
+          <img src="./images/program_icon_05.png" alt="arrival" />
+          <h3>Arrival</h3>
+          <p>
+            Watch our extraterrestrial friends arrive from different parts of
+            the universe
+          </p>
+        </div>
+        <div class="card">
+          <img src="./images/program_icon_02.png" alt="greeting" />
+          <h3>Greeting</h3>
+          <p>
+            Join us in for a warm welcome to our sentient partners from
+            different galaxies
+          </p>
+        </div>
+        <div class="card">
+          <img src="./images/program_icon_03.png" alt="exchange" />
+          <h3>Ideologic Exchange</h3>
+          <p>
+            We will learn the basics of communication with different species,
+            thanks to specialized interpreters
+          </p>
+        </div>
+        <div class="card">
+          <img src="./images/program_icon_04.png" alt="technology" />
+          <h3>Technology Interchange</h3>
+          <p>
+            Our friends have agreed to share some advanced technology to further
+            the development of our civilization
+          </p>
+        </div>
+        <div class="card">
+          <img src="./images/program_icon_01.png" alt="speakers" />
+          <h3>Alien Speakers</h3>
+          <p>
+            Representatives of different civilizations will share their
+            experience in building long-term societal structures
+          </p>
+        </div>
+      </div>
+      <div class="btn-container">
+        <button class="join-conference-btn">
+          Join us @ --> <span>First Contact!</span>
+        </button>
+      </div>
+      <div class="see-program">SEE THE WHOLE EVENT</div>
+    </section>
+    <section class="speakers" id="speakers">
+      <h2>Featured Speakers</h2>
+      <hr />
+      <div class="speakers-cards">
+        <div class="featured-speakers grid"></div>
+        <button id="more">MORE<span>v</span></button>
+        <div class="important-speakers grid"></div>
+      </div>
+    </section>
+    <section class="organizers" id="organizers">
+      <h2>Organizers</h2>
+      <hr class="organizers-hr" />
+      <div class="organizers-wrapper">
+        <img
+          src="./images/sponsor-logos/china_national_space_administration_logo.png"
+          alt=""
+        />
```

Comment:
- [ ] [OPTIONAL] Please try to include an alternate text in every `img` tag to help with accessiblity.